src/ACEtk/interpretation/EL03/test/Bremsstrahlung.test.cpp

```diff
@@ -35,7 +35,16 @@ namespace{
       2.030812000000E-02, 2.325490000000E-02, 2.573999000000E-02,
       2.789619000000E-02, 2.980029000000E-02, 3.299527000000E-02,
       3.556744000000E-02, 4.025667000000E-02, 4.342175000000E-02,
-      4.738277000000E-02, 4.985606000000E-02, 5.130644000000E-02 }};
+      4.738277000000E-02, 4.985606000000E-02, 5.130644000000E-02,
+      5.254506000000E-02, 5.420055000000E-02, 5.523288000000E-02,
```

Comment:
Could move data below tests - will require declaring the reference variables first.